src/work-allocation/containers/task-manager-list/task-manager-list.component.ts

```diff
@@ -43,6 +45,25 @@ export class TaskManagerListComponent extends TaskListWrapperComponent implement
     super(ref, taskService, router, infoMessageCommService, sessionStorageService, alertService, caseworkerService, loadingService);
   }
 
+  /**
+   * Gets the private logged in user id
+   */
+   public get userId(): string {
+    return this.pUserId;
+  }
+
+  /**
+   * Gets the caseworker's location
+   */
+  public get caseworkerLocation(): Location {
+    return this.pCaseworkerLocation;
+  }
+
+  private pUserId: string;
+  // pCaseworkerLocation is the caseworker that sets the location of the location dropdown
+  // Note: Setter for caseworkerLocation may come in useful if the selected location needs to be set via the caseworker assigned to the task
+  public pCaseworkerLocation: Location;
```

Comment:
Why the prefix 'p' for this public property?

Reply:
fixed the issues